design/ceph/object/bucketpolicy.md

```diff
@@ -0,0 +1,97 @@
+---
+title: adding bucket policy for ceph object store
+target-version: release-1.4
+---
+
+# Feature Name
+Adding bucket policy support for ceph object store
+
+## Summary
+The bucket policy is the feature in which permissions for specific user can be set on s3 bucket. Read more about it from [ceph documentation](https://docs.ceph.com/docs/master/radosgw/bucketpolicy/)
```

Comment:
don't need 'The' here (so "Bucket policy is the...")--but later, "the bucket policy feature" does need "the" and is correct